tests/unit/TableTest.php

```diff
@@ -64,13 +57,7 @@ public function testCreateRecord(): void
 
     public function testUpdateRecord()
     {
-        $table = $this->database->createTable(
-            new TableSchema('test-update-record', [
-                new Column('id', ColumnType::STRING(), true),
-                new Column('first_name', ColumnType::STRING()),
-                new Column('last_name', ColumnType::STRING())
-            ])
-        );
+        $table = $this->createTestTable('test-update-record',false);
```

Comment:
In reference to one of my comments below, remove the false value
and instead, add an age field with a value to the Record declaration.